ext_mongodb.php

```diff
@@ -715,17 +720,43 @@ function __debugInfo() : array;
 {
 	use DenySerialization;
 
-	public function __construct(private string $code, private ?mixed $scope = NULL)
+	private $code;
+
+	public function __construct(string $code, private ?mixed $scope = NULL)
 	{
+		if ( strstr( $code, "\0" ) !== false )
+		{
+			throw new \MongoDB\Driver\Exception\InvalidArgumentException( "Code cannot contain null bytes" );
+		}
+		$this->code = $code;
```

Comment:
I just looked at [master](https://github.com/mongodb/mongo-hhvm-driver/blob/bbf2974c52e11640aea09ac7adc2cd9d95bf3536/ext_mongodb.php#L718), and it doesn't appear that this constructor was ever assigning `$code` or `$scope`. Was that a longstanding bug we didn't catch with tests?

In any event, I think you still need to assign `$scope` here.